apps/ledger-live-mobile/src/components/CustomImage/useResizedImage.tsx

```diff
@@ -0,0 +1,63 @@
+import { useEffect } from "react";
+import { manipulateAsync, SaveFormat } from "expo-image-manipulator";
+import { ImageBase64Data, ImageDimensions, ImageFileUri } from "./types";
+import { ImageResizeError } from "./errors";
+
+export type ResizeResult = ImageBase64Data & ImageDimensions;
+
+export type Params = Partial<ImageFileUri> & {
+  targetDimensions: ImageDimensions;
+  onError: (error: Error) => void;
+  onResult: (res: ResizeResult) => void;
+};
+
+/**
+ * Component to resize an image.
+ *
+ * It:
+ *  - takes as an input an image file URI & target dimensions as an input
+ *  - renders nothing
```

Comment:
Nice hook :) just maybe update the comment to say that it's a hook now, so people don't get confused ;)

Reply:
thanks! done 👍